server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const cors = require('cors');
+const axios = require('axios');
+require('dotenv').config();
+
+const app = express();
+app.use(express.json());
+app.use(cors({ origin: '*' }));
```

Comment:
Suggest to have more specific whitelist of allowed origins.

Reply:
Thanks, fixed.